app/helpers/ems_cluster_helper/textual_summary.rb

```diff
@@ -254,6 +254,16 @@ def textual_aggregate_disk_capacity
     {:label => "Total Disk Capacity", :value => number_to_human_size(@record.aggregate_disk_capacity.gigabytes, :precision => 2)}
   end
 
+  def textual_cinder_disk_usage
+    return nil unless @record.kind_of?(ManageIQ::Providers::Openstack::InfraManager::EmsCluster) && @record.name.include?("BlockStorage")
```

Comment:
This feels wrong.  There should probably be a method on cluster like `.block_storage?`.  Then the check is just to see if it `.respond_to?(:block_storage?)`

That would make this code generic enough that you don't need "cinder" and "swift" specifics in the UI code.

cc @Ladas Can you see if you can help out @rwsu here to try and make the code a little more generic, particularly from the UI side?